plugins/sslservices/sslservices.cpp

```diff
@@ -407,17 +398,17 @@ OPENSSL_API void OPENSSL_CALL cipherEncrypt(ICodeContext *ctx, size32_t & __lenR
 
         try
         {
-            int len = 0;
-            int ciphertextLen = 0;
+            size32_t len = 0;
```

Comment:
This is a case where using int is probably correct - because that is the type required by the api you are calling.  (technically the reinterpret cast is undefined behaviour.)

Reply:
Changed cases where the type is required by the api to use that type.